src/support_sphere/lib/data/models/person.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:equatable/equatable.dart';
+
+class Person extends Equatable {
+  const Person({
+    this.nickname,
+    required this.id,
+    this.profile,
+    this.givenName = '',
+    this.familyName = '',
+    this.isSafe = true,
+    this.needsHelp = true,
+  });
+
+  /// The current user's id.
+  /// This is a unique identifier for the user as reflected in the database.
+  final String id;
+
+  /// The current user's profile.
+  /// This is optional and may be null.
+  final Profile? profile;
+  
+  /// The current user's given name.
+  final String givenName;
+
+  /// The current user's family name.
+  final String familyName;
+
+  /// The current user's nickname.
+  final String? nickname;
+
+  /// The current user's safety status.
+  final bool isSafe;
+
+  /// The current user's help status.
+  final bool needsHelp;
+
+  @override
+  List<Object?> get props => [
+        id,
+        profile,
+        givenName,
+        familyName,
+        nickname,
+        isSafe,
+        needsHelp,
+      ];
+}
+
+class Profile extends Equatable {
+
+  const Profile({
+    required this.id,
+    required this.userName,
+  });
+
+  /// The current user's id, which matches the auth user id
+  final String id;
+
+  /// The current user's username.
+  final String userName;
```

Comment:
Do we need usernames? This feels like more of a NextDoor/Facebook/etc app where people will be uniquely identified internally by their id and displayed as given/family name

Reply:
We technically don't need to as this is currently not being used anyways.